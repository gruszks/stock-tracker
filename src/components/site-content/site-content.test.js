import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import SiteContent from './site-content';

it('renders correctly', () => {
  const { asFragment } = render(
    <SiteContent title="Site title">text</SiteContent>
  );

  expect(asFragment()).toMatchSnapshot();
});
