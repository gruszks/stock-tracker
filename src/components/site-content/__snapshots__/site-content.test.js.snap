// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <main
    class="site-contentstyles__Main-sc-1a8hehg-0 bXKRmc"
  >
    <section>
      <h2
        class="site-contentstyles__Title-sc-1a8hehg-1 fNHMsa"
      >
        Site title
      </h2>
      text
    </section>
  </main>
</DocumentFragment>
`;
